picard/ui/mainwindow/__init__.py

```diff
@@ -1574,29 +1170,30 @@ def update_actions(self):
                     and can_submit
                 ):
                     break
-        self.remove_action.setEnabled(can_remove)
-        self.save_action.setEnabled(can_save)
-        self.view_info_action.setEnabled(can_view_info)
-        self.analyze_action.setEnabled(can_analyze)
-        self.generate_fingerprints_action.setEnabled(have_files)
-        self.refresh_action.setEnabled(can_refresh)
-        self.autotag_action.setEnabled(can_autotag)
-        self.browser_lookup_action.setEnabled(can_browser_lookup)
-        self.play_file_action.setEnabled(have_files)
-        self.open_folder_action.setEnabled(have_files)
-        self.cut_action.setEnabled(have_objects)
-        if self.submit_cluster_action:
-            self.submit_cluster_action.setEnabled(can_submit)
-        if self.submit_file_as_recording_action:
-            self.submit_file_as_recording_action.setEnabled(have_files)
-        if self.submit_file_as_release_action:
-            self.submit_file_as_release_action.setEnabled(have_files)
-        files = self.get_selected_or_unmatched_files()
-        self.tags_from_filenames_action.setEnabled(bool(files))
-        self.similar_items_search_action.setEnabled(is_file or is_cluster)
-        self.track_search_action.setEnabled(is_file)
-        self.album_search_action.setEnabled(is_cluster)
-        self.album_other_versions_action.setEnabled(is_album)
+
+        self.action_enabled(MainAction.REMOVE, can_remove)
+        self.action_enabled(MainAction.SAVE, can_save)
+        self.action_enabled(MainAction.VIEW_INFO, can_view_info)
+        self.action_enabled(MainAction.ANALYZE, can_analyze)
+        self.action_enabled(MainAction.GENERATE_FINGERPRINTS, have_files)
+        self.action_enabled(MainAction.REFRESH, can_refresh)
+        self.action_enabled(MainAction.AUTOTAG, can_autotag)
+        self.action_enabled(MainAction.BROWSER_LOOKUP, can_browser_lookup)
+        self.action_enabled(MainAction.PLAY_FILE, have_files)
+        self.action_enabled(MainAction.OPEN_FOLDER, have_files)
+        self.action_enabled(MainAction.CUT, have_objects)
+        self.action_enabled(MainAction.SUBMIT_CLUSTER, can_submit)
+        self.action_enabled(MainAction.SUBMIT_FILE_AS_RECORDING, have_files)
+        self.action_enabled(MainAction.SUBMIT_FILE_AS_RELEASE, have_files)
+        self.action_enabled(MainAction.TAGS_FROM_FILENAMES, self._get_selected_or_unmatched_files())
+        self.action_enabled(MainAction.SIMILAR_ITEMS_SEARCH, is_file or is_cluster)
+        self.action_enabled(MainAction.TRACK_SEARCH, is_file)
+        self.action_enabled(MainAction.ALBUM_SEARCH, is_cluster)
+        self.action_enabled(MainAction.ALBUM_OTHER_VERSIONS, is_album)
+
+    def action_enabled(self, action_id, enabled):
```

Comment:
Would `enable_action` be better? `action_enabled` to me sounds like you would use it to query the enabled state of an action.

Reply:
Yes, and shorter by one character, done in https://github.com/metabrainz/picard/pull/2450/commits/048c22c28c99fddf1e5921937d180b9a8d65028a